packages/client/tests/functional/interactive-transactions/tests.ts

```diff
@@ -11,222 +11,221 @@ declare let newPrismaClient: NewPrismaClient<typeof PrismaClient>
 
 const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
 
-testMatrix.setupTestSuite(
```

Comment:
Changes in this file are to do with the fact that I removed the: 

```
skipDataProxy: {
      runtimes: ['node', 'edge'],
      reason: 'Interactive transactions are not supported with Data Proxy yet',
    },
```

And the indent has gone.

Reply:
(Hiding whitespace changes in GitHub PR file view hides this pretty well)